Surface_mesh_parameterization/include/CGAL/Surface_mesh_parameterization/MVC_post_processor_3.h

```diff
@@ -418,27 +386,25 @@ class MVC_post_processor_3
     // The other parts of A(i,j) and A(i,k) will be added when this function
     // is called from the neighboring faces of F_ijk that share the vertex i
 
-    // Compute: - tan(alpha / 2)
-    NT w_i_base = -1.0 * compute_w_ij_mvc(pi, pj, pk);
-
     // @fixme unefficient: lengths are computed (and inversed!) twice per edge
 
+    // Set w_i_base: - tan(alpha / 2)
+    const Point_2& p = pk;
+    const Point_2& q = pi;
+    const Point_2& r = pj;
+    const CGAL::Weights::internal::
+      Tangent_weight_wrapper<NT> tangent_weight(p, q, r);
```

Comment:
Why do we have wrapper in the name?   I saw that also earlier for Polyhedron.

Reply:
I use wrapper in the name because it is not a class that computes tangent weights and it is not the one that should be documented. It is a wrapper for a combination of certain weight functions in order to obtain weights required in this part of the code. Moreover, the same wrapper is used in the other part of this package for doing the same thing. So, now they are uniform and call the same thing. Another argument is that we may have later decide to put some weights in the documented class called `Tangent_weight`, but that name is already occupied by this internal wrapper. So, I added wrapper to the name.

Reply:
These wrappers are renamed now and moved to the corresponding weight headers.